pkg/options/options.go

```diff
@@ -74,6 +76,8 @@ func NewKubeRouterConfig() *KubeRouterConfig {
 		BGPGracefulRestartDeferralTime: 360 * time.Second,
 		EnableOverlay:                  true,
 		OverlayType:                    "subnet",
+		ClusterIPCIDR:                  "10.96.0.0/12",
```

Comment:
Is this a safe guess for all of the user's that use our daemonset's that we publish?

Reply:
Its default value kubernetes uses.